spec/02-integration/03-db/08-declarative_spec.lua

```diff
@@ -10,6 +10,11 @@ for _, strategy in helpers.each_strategy() do
     lazy_setup(function()
       local _
       _, db = helpers.get_db_utils(strategy)
```

Comment:
Can we simply set the parameter`skip_migrations` to `true` to bootstrap the database?

https://github.com/Kong/kong-ee/blob/master/spec/helpers.lua#L583